app/controllers/sessions_controller.rb

```diff
@@ -14,6 +17,9 @@ def create
     session[:user_id] = user.id
 
     url = session[:pre_login_destination] || organizations_path
+
+    session[:current_scopes] = user.github_client_scopes
+
     redirect_to url
   end
```

Comment:
@mastahyeti can you add `# rubocop:enable Metrics/LineLength` at the end of the method? Otherwise it disables it for the rest of the file